src/lib/utilities/Toast/stores.ts

```diff
@@ -28,8 +28,22 @@ function toastService() {
 		trigger: (toast: ToastSettings) =>
 			update((tStore) => {
 				const id: string = randomUUID();
+				// Apply Preset Color Styles
+				let classes = toast.classes ?? '';
+				// prettier-ignore
+				switch (toast.preset) {
+					// Theme
+					case('primary'): classes += '!bg-primary-500 text-on-primary-token'; break;
+					case('accent'): classes += '!bg-accent-500 text-on-accent-token'; break;
+					case('tertiary'): classes += '!bg-tertiary-500 text-on-tertiary-token'; break;
+					case('warning'): classes += '!bg-warning-500 text-on-warning-token'; break;
+					// Success/Error
+					case('success'): classes += '!bg-green-500 !text-black'; break;
+					case('error'): classes += '!bg-red-500 !text-white'; break;
+				}
```

Comment:
This could be simplified down to this:
```suggestion
				switch (toast.preset) {
					// Success/Error
					case('success'): classes += '!bg-green-500 !text-black'; break;
					case('error'): classes += '!bg-red-500 !text-white'; break;
					// Theme
					default: classes += `!bg-${toast.preset}-500 text-on-${toast.preset}-token`; break;
				}
```
Though it may not be as explicit as you'd like

Reply:
Huh, the tabs look wild in this suggested changes preview

Reply:
One of those "oh duh" moments. Implementing this too. Thanks!